<?xml version="1.0"?>
<launch>

  <node pkg="move_arm" type="move_arm" name="move_arm" output="screen">
    <param name="serial_port" value="/dev/siix_dsp"/>
    <param name="baud_rate" value="9600"/>
  </node>

  <node pkg="rosserial_python" type="serial_node.py" name="serial_node" output="screen">
    <param name="port" value="/dev/cutter_nano"/>
    <param name="baud" value="57600"/>
  </node>
  
</launch>
